### DISCUSSION TITLE
avatar.url_as?

### DISCUSSION THREAD

**Discussion by NavisGames** [nextcord/nextcord#246] [Q&A] [answered]
Did you guys removed 
**avatar.url_as**
or what is the new thing to do that?

**Comment by ooliver1** [comment, marked as answer]
Asset.with_format and with_size

**Reply by NavisGames** [reply]
thank you :)

**Comment by DenverCoder1** [comment]
It was changed in discord.py 2.0 mainly because having multiple user avatars is now supported.

See [Migrating to 2.0](https://nextcord.readthedocs.io/en/latest/migrating_2.html#asset-changes) for full details.

```py
# previously str(avatar_url)
user.display_avatar.url

# previously str(avatar_url_as(size=128))
user.display_avatar.with_size(128).url

# previously str(avatar_url_as(size=128, static_format="png"))
user.display_avatar.replace(size=128, static_format="png").url
```

**Reply by NavisGames** [reply]
thx :)
